src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1250,6 +1250,7 @@ protected static void VerifyFlowGraphForTest<TSyntaxNode>(CSharpCompilation comp
         {
             var defaultRefs = useLatestFrameworkReferences ? s_latestOperationReferences : s_defaultOperationReferences;
             var references = additionalReferences == null ? defaultRefs : additionalReferences.Concat(defaultRefs);
+            parseOptions = parseOptions?.WithFlowAnalysisFeature() ?? TestOptions.RegularWithFlowAnalysisFeature;
```

Comment:
Perhaps we should add this to VB as well now?

Reply:
Yes, as soon as we have someone start to write VB tests.